resources/js/provider.js

```diff
@@ -3,7 +3,11 @@
     var callbackId = 0;
     var callbacks = {};
 
-    var bridgeSend = function (data) {
+    // TODO var is not necessary here
+    // if no var is prepended, this will be treated 
+    // as a global window.bridgeSend variable
+    // which is referenced from webview.js
+    bridgeSend = function (data) {
```

Comment:
good catch! but probably we should move provider js code here instead of making this method global ? im not sure why provider.js is separate file

Reply:
@flexsurfer an even better catch! :) Merged.